docs/software/programming.md

```diff
@@ -3,9 +3,11 @@
 ## Overview
 
 - [**Python - High-level interpreted programming language**](#python)
-- [**Docker - Build, ship, and run distributed applications**](#docker)
-- [**Docker Compose - Run multi-container applications on Docker**](#docker-compose)
-- [**Portainer - Docker container management**](#portainer)
+- [**Docker -  Create, deploy, and run applications using containers**](#docker)
+- [**Docker Compose - Define and run multi-container Docker applications**](#docker-compose)
+- [**Portainer - Lightweight management UI, managing your Docker host or Swarm cluster**](#portainer)
+
+## Install
```

Comment:
I think a new header is a bit much here. It's a one-liner hint (though expandable) only, and especially the below "Return to the Optimised Software list" fits well/better into the "Overview" header.